beacon-chain/params/BUILD.bazel

```diff
@@ -4,5 +4,8 @@ go_library(
     name = "go_default_library",
     srcs = ["config.go"],
     importpath = "github.com/prysmaticlabs/prysm/beacon-chain/params",
-    visibility = ["//beacon-chain:__subpackages__"],
+    visibility = [
+        "//beacon-chain:__subpackages__",
+        "//validator:__subpackages__",
```

Comment:
This shouldn't be allowed. Why are we exposing this to the other project?

Reply:
It doesn't look like you are using this, can you revert this change?

Reply:
Yeah sorry, I forgot to remove this